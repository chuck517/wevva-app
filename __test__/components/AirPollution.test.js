import React from 'react';
import renderer from 'react-test-renderer';
import AirPollution from '../../components/AirPollution';

beforeEach(() => {
  fetch.resetMocks();
});

it('renders without crashing', () => {
  const rendered = renderer.create(<AirPollution />).toJSON();
  expect(rendered).toBeTruthy();
});

it('test against snapshot', () => {
  const tree = renderer.create(<AirPollution />).toJSON();
  expect(tree).toMatchSnapshot();
});