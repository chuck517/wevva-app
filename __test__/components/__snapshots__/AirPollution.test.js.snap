// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AirPollution /> renders correctly across screens 1`] = `<View />`;

exports[`test against snapshot 1`] = `<View />`;
